Build/Build.cs

```diff
@@ -302,9 +302,48 @@ from framework in supportedFrameworks
             ReportTestOutcome(projects.Select(p => $"*{p.Name}*.trx").ToArray());
         });
 
+    Target TestingPlatformFrameworks => _ => _
```

Comment:
I would probably rename the old `TestFrameworks` `Target` to e.g. `LegacyTestPlatformFrameworks`, add a new `Target` named `TestFrameworks` and just add two dependents to this targets: `TestingPlatformFrameworks` and `LegacyTestPlatformFrameworks` like the `Target` `UnitTests`.

Reply:
Sure thing. Although shall I call them VSTest instead? Not sure they'll appreciate being called legacy, and they won't be going anywhere any time soon 🤣

Reply:
Haha.. true

Reply:
Made the change. Does that work? I simply added a new target with a dependency on the other two.

Reply:
Yeah.. seems right

You could check the dependency graph by running `nuke --plan` in the projects directory.